presidio-analyzer/setup.py

```diff
@@ -36,6 +36,7 @@
         "tldextract",
         "pyyaml",
         "phonenumbers>=8.12",
+        "validators"
     ],
```

Comment:
Don't forget to remove this

Reply:
I think that it's used?

Reply:
Oh, I see that you've decided to not use it after all? 
In that case also the from the pipfile

Reply:
Done